loki.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+set -Eeo pipefail
+trap cleanup SIGINT SIGTERM ERR
+
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
+
+usage() {
+  cat <<EOF
+Usage: $(basename "${BASH_SOURCE[0]}") [-h] [-v] [-t target] args
+
+Assists with running the Storybook Loki visual regression tests. Items in brackets
+are optional. Must have docker running and able to start up instances.
+
+Available options:
+
+args            One of: update/test/approve
+-h, --help      Print this help and exit
+-v, --verbose   Print script debug info
+-t, --target    Optional, core is default, options are: healthcare/medicare
+
+EOF
+  exit
+}
+
+cleanup() {
+  trap - SIGINT SIGTERM ERR
+  pkill -f storybook
+}
+
+die() {
+  local msg=$1
+  local code=${2-1} # default exit status 1
+  msg "$msg"
+  exit "$code"
+}
+
+msg() {
+  echo >&2 -e "${1-}"
+}
+
+run_storybook() {
+
+  while :; do
+    case "${1-}" in
+      -h | --help) usage ;;
+      -v | --verbose) set -x ;;
+      -t | --target) 
+        export LOKI_FILE_PREFIX="${2-}"
+        export STORYBOOK_DS="${2-}"
+        shift
+        ;;
+      *) break ;;
+    esac
+    shift
+  done
```

Comment:
Does it make sense to pull this out of this function now that it is in charge of populating the env vars for both loki and storybook?

Reply:
I think we should leave it, in case someone wants to run it by itself on a new theme in the future before adding to scripts

Reply:
Oh wait i understand.. I'll just rename it to something more generic

Reply:
Sorry, I just meant bumping this while loop code up to the script level and out of this function

Reply:
Could have a `parse_args` and `run_storybook` function if you'd rather keep it in a function

Reply:
oh yes of course, good point.. updated